bigbluebutton-html5/imports/ui/components/presentation/presentation-uploader/component.jsx

```diff
@@ -301,15 +451,14 @@ class PresentationUploader  extends PureComponent {
           });
         })
         .catch((error) => {
-          notify(intl.formatMessage(intlMessages.genericError), 'error');
```

Comment:
Is it intentional that you don't notify the user on error?

Reply:
It was intentional. This message was removed in 2.2, I assumed it got back in from a merge https://github.com/bigbluebutton/bigbluebutton/pull/8682/files#diff-0623880e37757bf6c1308106d96577c1L290